subsys/net/lib/openthread/platform/platform-zephyr.h

```diff
@@ -17,6 +17,49 @@
 
 #include <openthread/instance.h>
 
+/**
+ * Event types used to control open thread.
+ *
+ */
+enum pending_events {
+	PENDING_EVENT_NONE,  /* General event for processing non-event based
+			      * logic.
+			      */
+	PENDING_EVENT_TX_STARTED, /* Radio has started transmitting */
```

Comment:
How about prefixing radio events with `PENDING_EVENT_RADIO_`? Or we could drop the `PENDING` prefix to keep names shorter.